### PR TITLE
Version 3.0.0

### PR DESCRIPTION
**Pull request by josebolos**
* Changelog update
* Documentation updates
* Updated migration guide

Please note this commit would usually contain the version change in the package.json file, but that was done in a previous commit.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 
 # Changelog
 
+## 3.0.0 (2019-07-04)
+
+  * Update pa11y to v5, which replaces Phantomjs with Headless Chrome (thanks @wilco42)
+  * Add new index page, useful when running webservice as a standalone process (thanks @rtshilston)
+  * Add additional debugging info
+  * Update dependencies (thanks @paazmaya and others)
```
Comment (joeyciechanowicz) [inline CHANGELOG.md]:
```suggestion
  * Update dependencies (thanks @paazmaya, @josebolos  and others)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+  * Bump required node version to v8 or greater
+  * Lots of bug fixes (thanks @joeyciechanowicz for this, and also for helping reviewing PRs and issues)
+  * Minor documentation updates
```
Comment (joeyciechanowicz) [inline CHANGELOG.md]:
```suggestion
  * Documentation updates (thanks to @josebolos for this)
```

Comment (josebolos) [reply CHANGELOG.md]: 🙇

**Comment by josebolos** [comment]
😊

**Pull request merged**